### PR TITLE
test results, a few basic tests, warnings, join test classes, move tests back

### PR DESCRIPTION
**Pull request by tgf**
@diogofscmariano @pamval please review

### CODE REVIEW

**Comment by pamval** [comment]
@diogofscmariano, review and merge, please

File: core/src/main/java/pt/webdetails/cda/cache/EHCacheQueryCache.java [hunk 1]
```diff
@@ -66,148 +65,130 @@ public TableModel getTable() {
...
       //TODO: ehcache 2.5 has ClassLoaderAwareCache
-      Thread.currentThread().setContextClassLoader(this.getClass().getClassLoader());
-      final Element element = cache.get(key);
-      if (element != null)
-      {
-        final TableModel cachedTableModel = (TableModel) ((CacheElement) element.getObjectValue()).getTable();
-        if (cachedTableModel != null)
-        {
-          if (logger.isDebugEnabled()) {
+      Thread.currentThread().setContextClassLoader(this.getClass().getClassLoader() );
```
Comment (diogofscmariano) [inline core/src/main/java/pt/webdetails/cda/cache/EHCacheQueryCache.java]: missing checkstyle here

File: core/src/main/java/pt/webdetails/cda/cache/EHCacheQueryCache.java [hunk 2]
```diff
@@ -66,148 +65,130 @@ public TableModel getTable() {
...
+      final Element element = cache.get( key );
+      if ( element != null ) {
+        final TableModel cachedTableModel = (TableModel) ( (CacheElement) element.getObjectValue() ).getTable();
+        if ( cachedTableModel != null ) {
+          if ( logger.isDebugEnabled() ) {
             // we have a entry in the cache ... great!
-            logger.debug("Found tableModel in cache. Returning");
+            logger.debug("Found tableModel in cache. Returning" );
```
Comment (diogofscmariano) [inline core/src/main/java/pt/webdetails/cda/cache/EHCacheQueryCache.java]: missing checkstyle here

File: core/src/main/java/pt/webdetails/cda/cache/HazelcastQueryCache.java [hunk 3]
```diff
@@ -135,140 +130,129 @@ private static void init()
...
       int nbrTimeouts = incrTimeouts();
-      checkNbrTimeouts(nbrTimeouts);
-      logger.error("Timeout " + getTimeout + " " +  timeoutUnit + " expired fetching from " + 
-          map.getName() + " (timeout#" + nbrTimeouts + ")" );
-    } catch (InterruptedException e) {
-      logger.error(e);
-    } catch (ExecutionException e) {
-      logger.error(e);
+      checkNbrTimeouts( nbrTimeouts );
+      logger.error("Timeout " + getTimeout + " " +  timeoutUnit + " expired fetching from "
```
Comment (diogofscmariano) [inline core/src/main/java/pt/webdetails/cda/cache/HazelcastQueryCache.java]: missing checkstyle

**Comment by diogofscmariano** [comment]
There are some files not having checkstyle applied at all. Since we are checking this we should do the same for every file

**Comment by diogofscmariano** [comment]
@tgf there are some test classes removed. were they rearranged?

**Comment by tgf** [comment]
@diogofscmariano
IterableParameterSqlIT was removed after DataAccessTest.testIterableParameters
CsvIT was mostly removed: some of its scope is handled by the export tests and DefaultStreamExporterTest, the rest seemed yet another run of the same sql used by 5 other cda files.
Other missing classes were grouped together.

File: core/src/main/java/pt/webdetails/cda/cache/monitor/CacheElementInfo.java [hunk 4]
```diff
@@ -85,57 +85,56 @@ public Long getDuration() {
...
-      queryInfo.put("duration", duration.longValue());
+    queryInfo.put( "parameters", parameters );
+
+    queryInfo.put( "rows", rows != null ? rows.intValue() : null );
+
+    // inserted
+    queryInfo.put( "inserted", insertTime );
+    queryInfo.put( "accessed", accessTime );
+    queryInfo.put( "hits", hits );
+    queryInfo.put("size", byteSize != null ? byteSize.longValue() : null );
```
Comment (diogofscmariano) [inline core/src/main/java/pt/webdetails/cda/cache/monitor/CacheElementInfo.java]: missing here

File: core/src/main/java/pt/webdetails/cda/cache/monitor/CacheElementInfo.java [hunk 5]
```diff
@@ -85,57 +85,56 @@ public Long getDuration() {
...
+      queryInfo.put( "duration", duration.longValue() );
     }
-    queryInfo.put("timeToLive", timeToLive);
-    
-    //use id to get table;
-    //identifier
-    String identifier = TableCacheKey.getTableCacheKeyAsString(key);
-    queryInfo.put("key", identifier);
-    
+    queryInfo.put("timeToLive", timeToLive );
```
Comment (diogofscmariano) [inline core/src/main/java/pt/webdetails/cda/cache/monitor/CacheElementInfo.java]: missing here

**Pull request merged**